.github/workflows/java.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           
       - name: Code Format Check
         run: |
-          export JAVA_HOME=${JAVA_HOME_11_X64}
+          export JAVA_HOME=${JAVA_HOME_8_X64}
```

Comment:
why change to JAVA 8?

Reply:
I haven't used features that higher than Java 8, and Java 8 have better compatibility with others graph systems which rely on Java 8. So maybe we should assume that environment is Java 8, and Java higher is compatible with lower version.

Reply:
the spark rely on java 11, so for CI unification and avoid confusion， I think 11 is better.